tokenizers/src/processors/bert.rs

```diff
@@ -34,20 +40,34 @@ impl PostProcessor for BertProcessing {
         }
     }
 
-    fn process(
+    fn process_encodings(
         &self,
-        mut encoding: Encoding,
-        pair_encoding: Option<Encoding>,
+        mut encodings: Vec<Encoding>,
         add_special_tokens: bool,
-    ) -> Result<Encoding> {
+    ) -> Result<Vec<Encoding>> {
         if !add_special_tokens {
-            return <dyn PostProcessor>::default_process(
-                encoding,
-                pair_encoding,
-                add_special_tokens,
-            );
+            return Ok(encodings);
         }
 
+        let (mut encoding, pair_encoding): (Encoding, Option<Encoding>) = match encodings.len() {
+            1 => (
+                encodings
+                    .pop()
+                    .ok_or(ProcessorError::InvalidEncodingsVecLength)?,
```

Comment:
nit: can you just do `pop().unwrap()` here because you've already checked for length `encodings.len() == 1` or `ok_or` would still be better?

Reply:
I am removing this in the follow up PR anyway so shouldn't matter.

Reply:
oh I see 😊